src/main/java/net/boomerangplatform/controller/ConfigurationController.java

```diff
@@ -0,0 +1,34 @@
+package net.boomerangplatform.controller;
+
+import java.util.List;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.security.access.annotation.Secured;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PutMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+import net.boomerangplatform.model.FlowSettings;
+import net.boomerangplatform.service.crud.ConfigurationService;
+
+
+
+@RestController
+@RequestMapping("/ci/settings/config")
```

Comment:
Can this just be `flow/settings` ? Is the /config necessary?

Reply:
updated